docs/python_api/examples/renderdoc/decode_mesh.rst

```diff
@@ -115,10 +115,10 @@ For normalised formats - :py:attr:`~renderdoc.CompType.UNorm` and :py:attr:`~ren
 
         # If the format needs post-processing such as normalisation, do that now
         if fmt.compType == rd.CompType.UNorm:
-            divisor = float((1 << fmt.compByteWidth) - 1)
+            divisor = float((2 ** (fmt.compByteWidth * 8)) - 1)
             value = tuple(float(i) / divisor for i in value)
         elif fmt.compType == rd.CompType.SNorm:
-            maxNeg = -(1 << (fmt.compByteWidth - 1))
+            maxNeg = -float(2 ** (fmt.compByteWidth * 8)) / 2
```

Comment:
This fix should also be applied to `decode_mesh.py`.

Reply:
done